config.go

```diff
@@ -43,6 +43,8 @@ type Kafka struct {
 	RetryMax       int `toml:"retry_max"`
 	RetryBackoff   int `toml:"retry_backoff_ms"`
 	RepartitionMax int `toml:"repartition_max"`
+
+	Compression string `toml:"compression"` // ("gzip", "snappy" or "none", default: "none")
```

Comment:
[super nit] "" - is one of the allowed values : )
https://github.com/rakutentech/kafka-firehose-nozzle/pull/19/files#diff-f9adf7b6d6af18fc4e79344d34b1f743R57